readme.md

```diff
@@ -82,7 +71,7 @@ Hello, Node!
 
 * **Question:** Describe what is happening in the above code.
 
-* **Your Answer:**
+* **Your Answer:** assign path to a pets.json file for reference
```

Comment:
There's a little more detail that could be added here. What's `__dirname`? Why use `path.join()`?